ui/src/style/variables.less

```diff
@@ -23,7 +23,7 @@
 @s2-text-default: #2e414f;
 @s2-text-light: #8c9296;
 @s2-border-light: #e0e0e0;
-@s2-influential-citation-orange: #dd913f;
+@s2-influential-citation-orange: #3f91dd48;
```

Comment:
Curious where this hex code comes from? I thought hex codes were 6 digits max? Also I think it might be best to not change s2-influential-citation-orange and instead create a new variable. Thoughts?

Reply:
Agreed with @zkriby's point, I think semantically it makes sense to create a new variable, and leave the `@s2-influential-citation-orange` be so that it can be continue to be used by other components.